test/Microsoft.ML.Functional.Tests/Common.cs

```diff
@@ -160,6 +161,76 @@ public static void AssertEqual(TypeTestData testType1, TypeTestData testType2)
             Assert.True(testType1.Ug.Equals(testType2.Ug));
         }
 
+        /// <summary>
+        /// Check that a <see cref="AnomalyDetectionMetrics"/> object is valid.
+        /// </summary>
+        /// <remarks>
+        /// TODO #2644: At times, AnomalyDetection.Evaluate will return a set of NaN metrics.
+        /// </remarks>
```

Comment:
Maybe would not add the TODO to a xml comment. I would suggest having it as a regular comment only.